FakeServer.Test/FakeServerSpecs.cs

```diff
@@ -352,7 +352,35 @@ public async Task GetItem_WithId_Found()
         public async Task GetItem_WithId_NotFound()
         {
             var result = await _fixture.Client.GetAsync($"api/users/100000");
-            Assert.Equal(System.Net.HttpStatusCode.NotFound, result.StatusCode);
+            Assert.Equal(HttpStatusCode.NotFound, result.StatusCode);
+        }
+
+        [Fact]
+        public async Task GetItem_WithId_StoredInJsonAs_Integer()
+        {
+            var getIntegerId = await _fixture.Client.GetAsync($"api/movies/300");
```

Comment:
I pushed the branch with your suggestions. It is the right way (I mean `push --force` a branch in peer review) to do it on GitHub?

Also can you tell me where the CHANGELOG is?

Reply:
No need to use force. Better just to push a new commit.

When I merge this to master I use squash so it will just add single commit. Then it doesn't matter how nicely commented commits were or how nice history this branch had.

CHANGELOG is in the root
https://github.com/ttu/dotnet-fake-json-server/blob/master/CHANGELOG.md

Reply:
Alright. At least the commits and messages may have helped you for the review.

I updated the CHANGELOG